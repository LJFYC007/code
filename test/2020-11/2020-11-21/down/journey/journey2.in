30
956 138
9829 974
6279 5
1565 740
8854 466
210 102
7108 909
1561 458
6097 700
858 705
3553 38
7767 198
2364 486
3384 147
799 402
8544 616
9358 633
8501 176
161 453
7256 110
9258 784
9312 543
978 101
1146 57
473 437
175 975
2916 995
5869 758
266 623
9565 263
2 21 810
20 8 1366
11 18 1254
27 10 837
3 5 430
25 29 654
20 5 1183
19 27 1099
7 27 1045
2 13 75
23 9 1207
24 25 852
26 20 1490
23 22 257
29 14 1404
22 11 711
4 21 604
6 19 611
3 13 1459
24 6 1190
13 19 1378
9 7 689
28 20 908
12 20 579
16 22 79
29 1 1413
30 28 1200
15 30 863
1 17 1341
