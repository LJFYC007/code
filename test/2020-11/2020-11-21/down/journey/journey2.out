14814848
13474856
13107379
0
10381745
14931227
12696547
0
8203247
0
404624
0
15704342
0
0
0
0
0
15112880
9537935
6403960
5687528
6079031
14926739
14905136
0
14397286
4726151
14856059
3498204
